### ISSUE TITLE
create link to zenodo

### ISSUE THREAD

**Issue by KrisThielemans** [UCL/STIR#735]
It would be great to upload STIR releases to Zenodo. This gives an automatic creation of a DOI, and would allow people to refer to the STIR version they're using and also gives recent contributors finally a DOI to cite.  See also https://guides.github.com/activities/citable-code/

An example configuration file that we use for SIRF is [here](https://github.com/SyneRBI/SIRF/blob/master/.zenodo.json). As you can see, it lists contributors in some order, together with affiliations.

The "problem" with STIR is of course that there are many many contributors over the years. This makes finding author affiliation I've changed affiliations about 6 times...) and order a bit of a nightmare (by default zenodo uses https://github.com/UCL/STIR/graphs/contributors, but for whatever reason that goes back only to 2013). Therefore I suggest
- leave affiliations off (but people who want them can put them in)
- use @casperdcl 's [git-fame](https://github.com/casperdcl/git-fame) to find order for the Zenodo file. (@casperdcl , can git-frame generate the .zenodo.json for us?)

**Comment by casperdcl** [comment]
> can `git-fame` generate the `.zenodo.json` for us?

no but it should be trivial... you could open an issue if you want.

P.S. I'd recommend `git fame -wMC` for ordering attempting to ignore whitespace & renames.

**Comment by casperdcl** [comment]
P.S. just added a short stub of how to automate this https://github.com/casperdcl/git-fame/tree/devel#zenodo-config:

```sh
git fame -wMC --format json \
  | jq -c '{creators: [.data[] | {name: .[0]}]}' \
  | sed -r -e 's/(\{"name")/\n    \1/g' -e 's/:/: /g'
```

**Comment by KrisThielemans** [comment]
should have been auto-closed by merging #748
